### DISCUSSION TITLE
This extension always creates component's stylesheet as css

### DISCUSSION THREAD

**Discussion by utamuratov** [cyrilletuzi/vscode-angular-schematics#920] [Q&A] [answered]
When I create a component with this extension, It always creates the stylesheet of the component on CSS. But according to my angular json setting it should create less file

**Comment by cyrilletuzi** [comment, marked as answer]
Hello,

Moved this to discussions, as it is not a bug, but part a feature request, and part working as intended.

1. Less styles

I added back `less` value for `style` option of the component schematic (I wrongly thought it was deprecated, but I confused it with Stylus). Available in version 6.4.0 (it may require 2 VS Code reloads, one for the extension to update, the second one for the update to be applied).

2. About angular.json settings

It is working as intended. Since version 6 of the extension, it uses its own configuration.

To give more context about why: angular.json was becoming too complex, with no stable and simple official API to extract it. It was becoming a burden for the extension to move forward with some new features.

But I put a lot of effort in a configuration helper to configure everything as you like (like the style extension) in just a click, and even copy settings from angular.json. This configuration helper has been proposed to you on your first generation with the new extension version.

It has also been made visible from the extension panel accessible from the Angular logo on the left sidebar.  Also, the information about the new configuration has been made visible in the changelog and inside the extension documentation.

I will add a pinned message in GitHub issues to increase awareness about this change.
